scripts/test-html.js

```diff
@@ -0,0 +1,77 @@
+const process = require("process");
+const { JSDOM } = require("jsdom");
+const path = require("path");
+const http = require("http");
+const static = require('node-static');
+
+const { dir, base: filename } = path.parse(process.argv[2]);
+
+console.log(`serving: ${dir}`);
+
+serveDirectory(dir).then(server => {
+  const url = makeURL(server.address(), filename);
+
+  console.log(`loading ${url}`);
+
+  JSDOM.fromURL(url, {
+    resources: "usable",
+    runScripts: "dangerously",
+  })
+    .then(waitComplete)
+    .finally(() => server.close());
+}).catch(err => {
+  console.error(err);
+  process.exit(1);
+});
+
+function waitComplete(dom) {
+  const boxes = dom.window.document.getElementsByClassName("test-box-header");
+
+  var errors = []
+  var completed = 0
```

Comment:
For consistency with the style of everything else (ES 2015 + semicolons):
```suggestion
  const errors = [];
  let completed = 0;
```